CODEOWNERS

```diff
@@ -0,0 +1,19 @@
+## CODEOWNERS for autoreview assigning in github
```

Comment:
There's already a similar list [here](https://github.com/RIOT-OS/RIOT/wiki/Maintainers). Would be nice to have this reflected in the `CODEOWNERS` file.

Reply:
Will copy those over.

Reply:
(mostly) done